Library/Formula/pjproject.rb

```diff
@@ -0,0 +1,19 @@
+class Pjproject < Formula
+  homepage "http://www.pjsip.org/"
+  url "http://www.pjsip.org/release/2.3/pjproject-2.3.tar.bz2"
+  sha256 "e7fa60a3b59424430145af90372282ca778449f7b68b77bb24a9cf75d94d5765"
+
+  depends_on "openssl"
+
+  def install
+    system "./configure", "--prefix=#{prefix}"
+    system "make", "dep"
+    system "make"
+    system "make", "install"
+    bin.install Dir.glob("pjsip-apps/bin/pjsua-*").first => "pjsua"
```

Comment:
It'll probably be relatively easy to inspect the source and work out exactly how that string is generated (and possibly extract it from a generated file too)